amazon-kinesis-client/src/main/java/software/amazon/kinesis/leases/ShardSyncTaskManager.java

```diff
@@ -140,7 +144,11 @@ public TaskResult executeShardSyncTask() {
         return metricCollectingTask.call();
     }
 
-    public boolean syncShardAndLeaseInfo() {
+    /**
+     * Cast a ShardSyncTask and return if the casting is successful.
+     * @return if the casting is successful.
+     */
+    public boolean castShardSyncTask() {
```

Comment:
Should this be callShardSyncTask? A bit confused by the word cast since that seems overloaded with class casting

Reply:
changed to submit*